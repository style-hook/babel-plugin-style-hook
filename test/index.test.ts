import * as babel from '@babel/core'
import plugin from '../src/index'

var example = `
const c  = useStyle \`
  color: blue;
  font-size: 30px;
  .p {
    color: \${GREEN};
  }
\`
`

test('works', () => {
  const { code } = babel.transform(example, { plugins: [plugin] })
  expect(code).toMatchSnapshot()
})
