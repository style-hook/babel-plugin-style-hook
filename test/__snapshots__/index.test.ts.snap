// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `"const c = useStyle\`color:blue;font-size:30px;.p{color:\${GREEN}}\`;"`;
